### PR TITLE
Updates ddclient to version 3.8.3.

### PR DESCRIPTION
**Pull request by sorcix**

### CODE REVIEW

File: Library/Formula/ddclient.rb [hunk 1]
```diff
@@ -3,8 +3,8 @@
 class Ddclient < Formula
   desc "Update dynamic DNS entries"
   homepage 'http://sourceforge.net/p/ddclient/wiki/Home'
-  url 'https://downloads.sourceforge.net/project/ddclient/ddclient/ddclient-3.8.2/ddclient-3.8.2.tar.bz2'
-  sha1 '8c887c06a580773c48a1eaced82a08c3077e1325'
+  url 'https://downloads.sourceforge.net/project/ddclient/ddclient/ddclient-3.8.3/ddclient-3.8.3.tar.bz2'
+  sha1 '8668d8828a74ef4e7bca90890d7bbe414c37c3ff'
```
Comment (mikemcquaid) [inline Library/Formula/ddclient.rb]: Could you update this to use `sha256`? Thanks!

Comment (sorcix) [inline Library/Formula/ddclient.rb]: Ofcourse, I was not aware there is sha256 support.

File: Library/Formula/ddclient.rb [hunk 2]
```diff
@@ -3,8 +3,8 @@
 class Ddclient < Formula
   desc "Update dynamic DNS entries"
   homepage 'http://sourceforge.net/p/ddclient/wiki/Home'
-  url 'https://downloads.sourceforge.net/project/ddclient/ddclient/ddclient-3.8.2/ddclient-3.8.2.tar.bz2'
-  sha1 '8c887c06a580773c48a1eaced82a08c3077e1325'
+  url 'https://downloads.sourceforge.net/project/ddclient/ddclient/ddclient-3.8.3/ddclient-3.8.3.tar.bz2'
+  sha256 '5f2546687faf82a44c1addf058ef23bd29efba163c7fc1e9660f236aa092e9e51d301320cfdb72b78f2e65860386c30d85773d791f7c298d4ce42ba7dcdb4d2d'
```
Comment (mikemcquaid) [inline Library/Formula/ddclient.rb]: This is too long: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/29369/version=yosemite/testReport/junit/brew-test-bot/yosemite/install_ddclient/

Comment (DomT4) [inline Library/Formula/ddclient.rb]: Yeah, you've used a SHA512 rather than SHA256 here.

**Pull request closed without merging**